Marlin/language_en.h

```diff
@@ -508,5 +508,8 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
-
+#define MSG_CHANGE_FILAMENT                 "Filament"
+#define MSG_REMOVE_FILAMENT                 "Remove"
+#define MSG_INSERT_FILAMENT                 "Insert"
+#define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"
 #endif // LANGUAGE_EN_H
```

Comment:
@Blue-Marlin  i only have pasted my 4 variables at the end of the file i havent done any ifndef. ALso o see no warnings and i have done this 4 lines in pt-br file also, so i see it at my language.

@thinkyhead it was for me or for @Blue-Marlin? The question about sacarsm